src/DevBetterWeb.Core/ValueObjects/BillingDetails.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using CSharpFunctionalExtensions;
+
+namespace DevBetterWeb.Core.ValueObjects
+{
+  public class BillingDetails : ValueObject
+  {
+    public string Message { get; }
```

Comment:
Add methods to create messages for member activity and for admin activity reports.